FinniversKit/Sources/Recycling/GridViews/AdRecommendations/Cell/JobAdRecommendationCell.swift

```diff
@@ -218,7 +218,7 @@ extension JobAdRecommendationCell: AdRecommendationConfigurable {
         loadingColor = .backgroundSubtle
 
         containerView.accessibilityLabel = model?.accessibilityLabel
-        favoriteButton.accessibilityLabel = model?.favoriteButtonAccessibilityLabel
```

Comment:
Is the `accessibilityLabel` being set somewhere else?

Reply:
Good catch, I lost it during merging
Put back it here: https://github.com/finn-no/FinniversKit/pull/1398/files#diff-9c2c5679975471f2eb787c6436339de8efe689425e440bd4e0e5a92a9e5ac503R24